modules/veteran/spec/workers/veteran/vso_reloader_spec.rb

```diff
@@ -15,8 +15,8 @@
     it 'should reload data from pulldown' do
       VCR.use_cassette('veteran/ogc_poa_data') do
         Veteran::VsoReloader.new.perform
-        expect(Veteran::Service::Representative.count).to eq 152
-        expect(Veteran::Service::Organization.count).to eq 3
+        expect(Veteran::Service::Representative.count).to eq 23_361
+        expect(Veteran::Service::Organization.count).to eq 91
```

Comment:
Does are test cassette really have 23k records? I think that it to many and will make the test too slow. Previously with 9k records it was a minute just for the single test. Can we modify the test cassette to not import the whole data set?

Reply:
I'll see if I can strip down the VCR in some way.

Reply:
So all of the attempts I made at modifying the cassette to reduce the payload down even further were met with one error after another below (even when I tried to recalculate the payload hash after modifying it etc.

```1) Veteran::VsoReloader importer should reload data from pulldown
     Failure/Error: Veteran::VsoReloader.new.perform
     
     VCR::Errors::UnhandledHTTPRequestError:
     
     
       ================================================================================
       An HTTP request has been made that VCR does not know how to handle:
         POST https://www.va.gov/ogc/apps/accreditation/caexcellist.asp
     
       VCR is currently using the following cassette:
         - /home/michael/oddball/vets-api/spec/support/vcr_cassettes/veteran/ogc_poa_data.yml
           - :record => :once
           - :match_requests_on => [:method, :uri]
     
       Under the current configuration VCR can not find a suitable HTTP interaction
       to replay and is prevented from recording new requests. There are a few ways
       you can deal with this:
     
         * If you're surprised VCR is raising this error
           and want insight about how VCR attempted to handle the request,
           you can use the debug_logger configuration option to log more details [1].
         * You can use the :new_episodes record mode to allow VCR to
           record this new request to the existing cassette [2].
         * If you want VCR to ignore this request (and others like it), you can
           set an `ignore_request` callback [3].
         * The current record mode (:once) does not allow new requests to be recorded
           to a previously recorded cassette. You can delete the cassette file and re-run
           your tests to allow the cassette to be recorded with this request [4].
         * The cassette contains 1 HTTP interaction that has not been
           played back. If your request is non-deterministic, you may need to
           change your :match_requests_on cassette option to be more lenient
           or use a custom request matcher to allow it to match [5].
     
       [1] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/configuration/debug-logging
       [2] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/record-modes/new-episodes
       [3] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/configuration/ignore-request
       [4] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/record-modes/once
       [5] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/request-matching
       ================================================================================
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/vcr-3.0.3/lib/vcr/request_handler.rb:97:in `on_unhandled_request'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/vcr-3.0.3/lib/vcr/library_hooks/webmock.rb:129:in `on_unhandled_request'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/vcr-3.0.3/lib/vcr/request_handler.rb:24:in `handle'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/vcr-3.0.3/lib/vcr/library_hooks/webmock.rb:144:in `block in <module:WebMock>'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/stub_registry.rb:28:in `block in register_global_stub'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/request_pattern.rb:40:in `matches?'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/stub_registry.rb:58:in `block in request_stub_for'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/stub_registry.rb:57:in `each'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/stub_registry.rb:57:in `detect'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/stub_registry.rb:57:in `request_stub_for'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/stub_registry.rb:50:in `response_for_request'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/webmock-3.5.1/lib/webmock/http_lib_adapters/net_http.rb:79:in `request'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:82:in `perform_request'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:40:in `block in call'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:87:in `with_net_http_connection'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/adapter/net_http.rb:32:in `call'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/request/url_encoded.rb:15:in `call'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/rack_builder.rb:139:in `build_response'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/connection.rb:377:in `run_request'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/faraday-0.9.2/lib/faraday/connection.rb:177:in `post'
     # ./modules/veteran/app/workers/veteran/base_reloader.rb:23:in `fetch_data'
     # ./modules/veteran/app/workers/veteran/vso_reloader.rb:45:in `reload_claim_agents'
     # ./modules/veteran/app/workers/veteran/vso_reloader.rb:16:in `reload_representatives'
     # ./modules/veteran/app/workers/veteran/vso_reloader.rb:8:in `perform'
     # ./modules/veteran/spec/workers/veteran/vso_reloader_spec.rb:17:in `block (4 levels) in <top (required)>'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/vcr-3.0.3/lib/vcr/util/variable_args_block_caller.rb:9:in `call_block'
     # /home/michael/.rvm/gems/ruby-2.4.5/gems/vcr-3.0.3/lib/vcr.rb:189:in `use_cassette'
     # ./modules/veteran/spec/workers/veteran/vso_reloader_spec.rb:16:in `block (3 levels) in <top (required)>'```

If you have another way to do it let me know but otherwise I've got no way so far of pulling off a file reduction.